### PR TITLE
feat(login): login css added

### LINKED ISSUE #27: [FEATURE] Add styles for Login page
## What

Add styles according to design: https://www.figma.com/file/LSKPBAnOkYg5odtGwC0yIb/MeetingBook-Design-Kit?node-id=0%3A10

### PR DESCRIPTION
**Pull request by NatalliaMil**
closes #27

### CODE REVIEW

File: web/app/login.html [hunk 1]
```diff
@@ -25,21 +27,48 @@ <h1  class="mdc-typography--headline1">Login</h1>
...
       </span>
-      <input type="email" class="mdc-text-field__input" aria-labelledby="my-label-id">
-    </label><br>
-    <label class="mdc-text-field mdc-text-field--outlined">
+      <input
+        type="email"
+        class="mdc-text-field__input"
+        aria-labelledby="my-label-id"
+      /> </label
+    ><br />
```
Comment (glebhg) [inline web/app/login.html:35]:
```suggestion
    >
```

File: web/app/login.html [hunk 2]
```diff
@@ -25,21 +27,48 @@ <h1  class="mdc-typography--headline1">Login</h1>
...
+      <i
+        class="
+          material-icons
+          mdc-text-field__icon mdc-text-field__icon--trailing
+        "
+        tabindex="0"
+        role="button"
+        >visibility</i
+      > </label
+    ><br />
```
Comment (glebhg) [inline web/app/login.html:64]:
```suggestion
    >
```

File: web/app/css/styles.css [hunk 3]
```diff
@@ -1,3 +1,55 @@
+:root {
+  --mdc-theme-primary: #d55a5a;
+  --color-white: #fff;
+}
+
 body {
-  background-color: aliceblue;
+  background: aliceblue url('/images/loginbackground.svg') no-repeat 50% 50%;
```
Comment (glebhg) [inline web/app/css/styles.css:7]: It is better to have all colors as custom properties, so you can move aliceblue there.

File: web/app/login.html (hunk 1 above)
Comment (glebhg) [reply web/app/login.html:35]: Please, don't use <br> cause it is bad for accessibility. You can always replace <br> with margin-bottom for label :)

**Review by glebhg — CHANGES_REQUESTED** [review]
Amazing work, just few issues to fix

**Review by glebhg — APPROVED** [review]

**Review by yogahub — APPROVED** [review]

**Pull request merged**